content/tags/control_flow_tags.md

```diff
@@ -0,0 +1,100 @@
+# Control Flow Tags
+
+Control Flow tags determine which block of code should be executed based on different conditions.
+
+1. TOC
+{:toc}
+
+## if
+
+Executes a block of code only if a certain condition is met.
+
+#### Input
+
+~~~ liquid
+  {% if rental.title == 'Apartment 051' %}
+      This Apartment is with a pool!
```

Comment:
Hmm, `comes with a swimming pool`? not sure.
I would drop the `!` everywhere though